applications/vessel-history/src/features/profile/components/Info.tsx

```diff
@@ -201,6 +201,14 @@ const Info: React.FC<InfoProps> = (props): React.ReactElement => {
                   )}
                 </div>
               </div>
+              {vessel.iuuStatus !== undefined && 
+                <InfoField
+                  vesselName={vessel.shipname ?? DEFAULT_EMPTY_VALUE}
+                  label={VesselFieldLabel.iuuStatus}
+                  value={t(`vessel.iuuStatusOptions.${vessel.iuuStatus}` as any, t('common.unknown', 'unknown'))}
```

Comment:
In this case if we don't have the translation for that IUU status we could probably display the IUU status code , [follow up on the comment I added to the ticket](https://globalfishingwatch.atlassian.net/browse/VV-132?focusedCommentId=13395) about this.

Reply:
I fixed both comment in one place